components/automate-cli/cmd/chef-automate/automateHAAwsDeployment.go

```diff
@@ -205,7 +213,10 @@ func (a *awsDeployment) validateConfigFields() *list.List {
 func (a *awsDeployment) validateEnvFields() *list.List {
 	errorList := list.New()
 	if len(a.config.Aws.Config.Profile) < 1 {
-		errorList.PushBack("Invalid or empty aws profile name")
+		isIAMRole, _ := a.isIamRolePresent()
```

Comment:
Is attaching the IAM role necessarily mean having admin access to create resources?
If yes, we are good here. 
Once check with Punit.
If no, can we check that as well

Reply:
Resolving this, as this could be an enhancement